.eslintrc.json

```diff
@@ -234,7 +264,13 @@
                     "off"
                 ],
                 "no-await-in-loop": [
-                    "off"
+                    "error"
+                ],
+                "no-return-await": [
+                    "error"
```

Comment:
Again, too limiting without any good advantages. For example:
```
async function findRows () {
    let rows = await db.find(),
         transformed = rows.map(x => x * 2);
    
    return await this.calculate(transformed);
}
```
A decent case. I don't see any reasons why that's bad.

Reply:
Agree with Artem. Any examples?

Reply:
I agree that it would be easier for me to then use the result of that findRows function rather than dealing with another promise :D

But as rule suggests, if returning await, then this current function is kept in stack until the promise resolves which kind of pollutes the stack and has a minor performance hit (https://eslint.org/docs/rules/no-return-await)

But I could not find any hard evidence of that and benchmarks that would prove performance hit. 
So I guess we can turn that off for the sake of usability

Reply:
Oh, actually
```
Since the return value of an async function is always wrapped in Promise.resolve, return await doesn’t actually do anything except add extra time before the overarching Promise resolves or rejects.
```

Reply:
http://hassansin.github.io/Why-return-await-Is-a-Bad-Idea

Reply:
Oh, how didn't I think about that. Then yes, let's keep it.